reframe/core/exceptions.py

```diff
@@ -243,7 +243,8 @@ def format_user_frame(frame):
         return 'sanity error: %s' % exc_value
 
     if isinstance(exc_value, PerformanceError):
-        return 'performance error: %s' % exc_value
+        exc = exc_value.__cause__ or exc_value
```

Comment:
The problem with `: :` is more general and should be treated inside the `ReFrameError`.